_posts/2024/2024-02-06-about-caching-in-eo.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+date: 2024-02-06
+title: "Build cache in EO and other build systems"
+author: Alekseeva Yana
+---
+
+
+## Introduction 
+In [EO](https://github.com/objectionary/eo) a caching is used to speed up program execution.
+While developing [EO](https://github.com/objectionary/eo) we found a caching 
+[error](https://github.com/objectionary/eo/issues/2790) in `eo-maven-plugin`
+for EO version `0.34.0`. The error occurred because the cache was searched for the needed file using 
+a comparison of compilation time and caching time.
+This is not the most reliable verification method,
+because caching time does not have to be equal to compilation time.
+[Unit tests](https://github.com/objectionary/eo/pull/2749) were written to show that the
+cache does not work correctly. Additionally, reading a file was necessary to obtain a program name 
```

Comment:
@Yanich96 [Question] To be honest, I don't think we need such details in this post. They only confuse a reader. Maybe it's better to omit them? What do you think?